src/services/emailService.test.js

```diff
@@ -0,0 +1,30 @@
+import emailService from "./EmailService";
+import isValidEmail from "../components/helpers/validators/email-address-validator";
+
+describe("check emailService", () => {
+  const error = "Invalid email address";
+  it("Email is not empty", () => {
+    const values = {
+      email: "example@gmail.com",
+    };
+    expect(emailService(values)).toBe(true);
+  });
+  it("Correct email", () => {
+    const correctEmail = isValidEmail("example@gmail.com");
+    expect(correctEmail).toStrictEqual({});
+  });
+  it("Email without at(@)", () => {
+    const inCorrectEmail = isValidEmail("examplegmail.com");
+    expect(inCorrectEmail).toStrictEqual({ email: error });
+  });
+  it("Email without address", () => {
+    const inCorrecAddress = isValidEmail("examplegmailcom");
+    expect(inCorrecAddress).toStrictEqual({ email: error });
+  });
```

Comment:
Why you test isValidEmail func in this file? You need to create tests only for emailService. Remove them or create new file for this tests.